src/ussd_higherhealth_healthcheck.js

```diff
@@ -224,10 +225,767 @@ go.app = (function () {
+                "Howard",
+                "Medical School",
+                "Pietermaritzburg",
+                "Westville"
+        ],
+            "University of Limpopo (UL)":[
+                "Turfloop Campus"
+        ],
+            "University of Mpumalanga (UMP)":[
+                "Mbombela",
+                "Siyabuswa"
+
+        ],
+            "University of Pretoria (UP)":[
+                "GIBS",
+                "Groenkloof",
+                "Hatfield (Main Campus)",
+                "Hillcrest",
+                "Mamelodi",
+                "Onderstepoort",
+                "Prinshof"
+
+        ],
+            "University of Venda (UNIVEN)":[
+                "Thohoyandou"
+
+        ],
+            "University of Western Cape (UWC)":[
+                "Bellville"
+
+        ],
+            "University of Zululand (UNIZULU)":[
+                "KwaDlangezwa",
+                "Richards Bay"
+
+        ],
+            "University of the Free State (UFS)":[
+                "Main Campus",
+                "Qwaqwa Campus",
+                "South Campus"
+
+        ],
+            "University of the Witwatersrand (WITS)":[
+                "Braamfontein"
+
+        ],
+            "Vaal University of Technology (VUT)":[
+                "Ekurhuleni",
+                "Main Campus (Vanderbijlpark)",
+                "Secunda"
+
+        ],
+            "Vhembe":[
+                "Central Office",
+                "Makwarela",
+                "Mashamba",
+                "Mavhoi",
+                "Musina",
+                "Shingwedzi",
+                "Thengwe",
+                "Tshisimani"
+
+        ],
+            "Vuselela":[
+                "Central Office",
+                "Jouberton",
+                "Klerksdorp",
+                "Matlosana",
+                "Potchefstroom",
+                "Taung"
+
+        ],
+            "Walter Sisulu University (WSU)":[
+                "Buffalo City",
+                "Butterworth (Ibika)",
+                "Mthatha",
+                "Queenstown (Masibulele)"
+
+        ],
+            "Waterberg":[
+                "Business",
+                "Central Office",
+                "Engineering",
+                "IT&C",
+                "Thabazimbi"
+
+        ],
+            "West Coast":[
+                "Atlantis",
+                "Central Office",
+                "Citrusdal",
+                "Malmesbury",
+                "Vredenburg",
+                "Vredendal"
+
+        ],
+            "Western":[
+                "Carletonville",
+                "Central Office",
+                "Krugersdorp",
+                "Krugersdorp West",
+                "Randfontein"
+
+        ],
+        };
+      var selected_prov = self.im.user.answers.state_province;
+      var selected_uni = self.im.user.answers.state_university;
+      var choices = [];
+      var campus_list = [];
+      if (selected_uni !== 'UNISA') campus_list = campuses_by_university[selected_uni];
```

Comment:
I think there's a misunderstanding here. It's not just UNISA that has campuses in multiple provinces, that was just the example I chose. We need to do this for all campuses, not just UNISA campuses.

Reply:
👍